$OpenBSD: patch-apertium_apy_handlers_list_modes_py,v 1.1 2021/01/23 08:08:55 daniel Exp $

fix the build with tornado6

backport of commit 15cfa135 from upstream

Index: apertium_apy/handlers/list_modes.py
--- apertium_apy/handlers/list_modes.py.orig
+++ apertium_apy/handlers/list_modes.py
@@ -1,11 +1,11 @@
-import tornado.web
+import tornado.gen
 
 from apertium_apy.handlers.base import BaseHandler
 from apertium_apy.utils import to_alpha2_code
 
 
 class ListHandler(BaseHandler):
-    @tornado.web.asynchronous
+    @tornado.gen.coroutine
     def get(self):
         query = self.get_argument('q', default='pairs')
 
